1305.py

```diff
@@ -0,0 +1,35 @@
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, val=0, left=None, right=None):
+#         self.val = val
+#         self.left = left
+#         self.right = right
+class Solution:
+    def getAllElements(self, root1: TreeNode, root2: TreeNode) -> List[int]:
+        def pushleft(node: TreeNode, stack: List[TreeNode]):
+            while node:
```

Comment:
This `pushleft` template was carefully designed to automatically take care of the `node == None` case using the `while` statement.